Gems/Atom/Feature/Common/Assets/ShaderLib/Atom/Features/PBR/Surfaces/StandardSurface.azsli

```diff
@@ -15,8 +15,14 @@
 
 class Surface
 {
+
+#if ENABLE_CLEAR_COAT
     ClearCoatSurfaceData clearCoat;
+#endif
+
+#if ENABLE_TRANSMISSION
     TransmissionSurfaceData transmission; // This is not actually used for Standard PBR, but must be present for common lighting code to compile
+#endif
```

Comment:
Can we just delete this? StandardSurface is only used in the "StandardPbr" category of materials, and so will never support transmission, right?

Reply:
Will remove